.prettierrc.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
This file shouldnt be needed, we specify the config via the `prettier` property in package.json

Reply:
Removed